### PR TITLE
disable progress bars when running the tests to avoid messing up test suite output

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

File: test/framework/utilities.py [hunk 1]
```diff
@@ -137,6 +137,11 @@ def setUp(self):
 
         init_config()
 
+        # disable progress bars when running the tests,
+        # since it messes up with test suite progress output when test installations are performed
```
Comment (akesandgren) [inline test/framework/utilities.py:141]: "since it messes with test suite progress output"
or
"since it messes up test suite progress output"

**Review by akesandgren — CHANGES_REQUESTED** [review]

File: test/framework/utilities.py (hunk 1 above)
Comment (boegel) [reply test/framework/utilities.py]: fixed in d5ccae925

**Review by akesandgren — APPROVED** [review]

**Comment by boegel** [comment]
Hmm, that failing test on Python 3.9.7 is very strange...
Looks like that could be a regression in that specific Python version?!

**Comment by boegel** [comment]
> Hmm, that failing test on Python 3.9.7 is very strange...
> Looks like that could be a regression in that specific Python version?!

Not really a regression I guess, but indeed due to a change in Python 3.9.7, fixed in #3840

**Review by akesandgren — APPROVED** [review]

**Comment by akesandgren** [comment]
Going in, thanks @boegel!

**Pull request merged**